static/app/views/issueDetails/groupSidebar.tsx

```diff
@@ -145,7 +149,12 @@ export default function GroupSidebar({
           />
         </SidebarSection.Title>
         <SidebarSection.Content>
-          <StyledAvatarList users={participants} avatarSize={28} maxVisibleAvatars={13} />
+          <StyledAvatarList
```

Comment:
Looks like you want to pass `typeAvatars="participants"` here so the overflow label says `n other participants` instead of `n other users`.

Side note: you didn't create the name, but `typeAvatars` seems like a bad name :/